### DISCUSSION TITLE
Errors while converting from wix3.14 to wix 4

### DISCUSSION THREAD

**Discussion by RAppels2022** [wixtoolset/issues#7027] [Questions] [answered]
While converting from 3.14 to wix 4 in VS2019, all the include files have disappeared but the errors come from those include files.

Build started...
1>------ Build started: Project: SyncExcelTables, Configuration: Release Any CPU ------
1>  SyncExcelTables -> D:\Source\repos\SyncExcelTables\SyncExcelTables\bin\Release\SyncExcelTables.exe
1>  Done Adding Additional Store
1>  Successfully signed: D:\Source\repos\SyncExcelTables\SyncExcelTables\bin\Release\SyncExcelTables.exe
1>  
2>------ Build started: Project: WixSetupProject, Configuration: Release x86 ------
2>D:\Source\repos\SyncExcelTables\WixSetupProject\Includes\CheckAdminRights.wxi(7): error WIX0005: The Package element contains an unexpected child element 'Condition'.
2>D:\Source\repos\SyncExcelTables\WixSetupProject\Includes\CheckFramework4.8.wxi(10): error WIX0005: The Package element contains an unexpected child element 'Condition'.
2>D:\Source\repos\SyncExcelTables\WixSetupProject\Includes\CheckOSversion7andHigher.wxi(2): error WIX0005: The Package element contains an unexpected child element 'Condition'.
2>Done building project "WixSetupProject.wixproj" -- FAILED.
StopOnFirstBuildError: Build cancelled because project "WixSetupProject" failed to build.
Build has been canceled.

Severity	Code	Description	Project	File	Line	Suppression State
Message		Could not find schema information for the attribute 'Message'.	Miscellaneous Files	D:\Source\repos\SyncExcelTables\WixSetupProject\Includes\CheckAdminRights.wxi	7	
Message		Could not find schema information for the element 'http://wixtoolset.org/schemas/v4/wxs:Condition'.	Miscellaneous Files	D:\Source\repos\SyncExcelTables\WixSetupProject\Includes\CheckAdminRights.wxi	7	
Error	WIX0005	The Package element contains an unexpected child element 'Condition'.	WixSetupProject	D:\Source\repos\SyncExcelTables\WixSetupProject\Includes\CheckAdminRights.wxi	7	
Error	WIX0005	The Package element contains an unexpected child element 'Condition'.	WixSetupProject	D:\Source\repos\SyncExcelTables\WixSetupProject\Includes\CheckFramework4.8.wxi	10	
Error	WIX0005	The Package element contains an unexpected child element 'Condition'.	WixSetupProject	D:\Source\repos\SyncExcelTables\WixSetupProject\Includes\CheckOSversion7andHigher.wxi	2	

What can  I do to fix this, or do I have to wait on the next preview version?

**Comment by robmen** [comment]
This looks like a [HeatWave](https://www.firegiant.com/products/) conversion issue. Would you mind opening an issue over at [HeatWave Support](https://github.com/firegiant/HeatWaveSupport/issues/new/choose) with details about how your .wxi files were listed in your .wixproj? I expect that information will be relevant for the FireGiant team to investigate.

**Comment by RAppels2022** [comment, marked as answer]
I have created a conversion issue at #https://github.com/firegiant/HeatWaveSupport/issues/

**Comment by rseanhall** [comment]
You don't have to wait for the next preview version. You should be able to run `wix.exe convert` on the .wxi files manually.
